content/fr/docs/contribute/intermediate.md

```diff
@@ -0,0 +1,834 @@
+---
+title: Contribution intermédiaire
+slug: intermediaire
+content_template: templates/concept
+weight: 20
+---
+
+{{% capture overview %}}
+
+Cette page assume que vous avez lu et compris les tâches dans le topic
```

Comment:
assume ? Change the verb please

Reply:
"assume" is a french verb but "suppose" works better thanks